src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> products;
+
+    public Storage() {
+        products = new HashMap<>();
+    }
+
+    public Map<String, Integer> getProducts() {
+        return products;
+    }
+
+}
```

Comment:
I think you can use in line declaration of map here and not use constructor and getter. But your approach is ok too, nobody cancels encapsulation.

Reply:
I'm not sure too, so let's wait for a mentor))

Reply:
actually this is a bit confusing because every time you create a new `Storage`, you reinitialize the map, which is unclear and shouldn't work like that, I would actually just make it 
`public static final Map<...> products = new hashMap<>()` without any getters etc. because we have another level of abstration for it in the form of `StorageService`